pkgs/development/haskell-modules/configuration-darwin.nix

```diff
@@ -117,6 +117,10 @@ self: super: ({
 
   yesod-bin = addBuildDepend darwin.apple_sdk.frameworks.Cocoa super.yesod-bin;
 
+  yesod-core = super.yesod-core.overrideAttrs (drv: {
+    __darwinAllowLocalNetworking = true;
```

Comment:
Could you add a comment in the code here why this is needed?

Also, could someone that knows Darwin do a reveiw of this PR?  @NixOS/darwin-maintainers

Reply:
Sure. I’ll also do that for the http2 PR.